CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # OverReact Changelog
 
+## [3.11.0](https://github.com/Workiva/over_react/compare/3.10.1...3.11.0)
+
+- [#633] Add `staticMeta`, `addUnconsumedProps`, and `addUnconsumedDomProps` to the `UiProps` component base class.
```

Comment:
Can we include something here about what these added APIs are accomplishing (supporting consumed props in function components)